src/components/AddActivityForm/AddActivityForm.spec.js

```diff
@@ -1,21 +1,25 @@
 import {render, screen} from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
-import ModalInput from './ModalInput';
+import AddActivityForm from './AddActivityForm';
 
-describe('ModalInput', () => {
+describe('AddActivityForm', () => {
   it('is an accessible form', () => {
-    render(<ModalInput />);
+    render(<AddActivityForm />);
 
     screen.getByRole('form');
     screen.getByLabelText('Add your Activity');
     screen.getByRole('button', {name: 'Add Activity'});
   });
 
-  it('calls onHandleSubmit on submit', async () => {
+  it('calls handleSubmit on submit', async () => {
     const user = userEvent.setup();
     const callback = jest.fn(e => e.preventDefault());
-    render(<ModalInput onHandleSubmit={callback} />);
+    const callbackSetActivities = jest.fn();
+    const callbackChange = jest.fn();
+    render(
+      <AddActivityForm handleChange={callbackChange} handleSubmit={callback} onSetActivities={callbackSetActivities} />
```

Comment:
The only prop your `AddActivityForm` has is `onSetActivities` the rest is not used anyways.
```suggestion
      <AddActivityForm onSetActivities={callbackSetActivities} />
```